tests/auth_tests/test_templates.py

```diff
@@ -29,15 +31,26 @@ def test_titles(self):
         self.assertContains(response, '<h1>Password reset sent</h1>')
 
         # PasswordResetConfirmView invalid token
-        response = PasswordResetConfirmView.as_view(success_url='dummy/')(request, uidb64='Bad', token='Bad')
+        # We need to add a SessionMiddleware, as password reset confirm view uses session
```

Comment:
Should I only update this one, or do you prefer all tests to be updated for consistency?